src/service/state.js

```diff
@@ -222,6 +222,25 @@ module.exports = (statePersistence, municipalityService, seService) => {
       };
     },
 
+    /**
+     * Request a given strategic ecosystem HF timeline data inside a state
+     * @param {Number} stateId state id
+     * @param {String} seType strategic ecosystem type
+     *
+     * @return {Object} Object of HF values through time
+     */
+
+    getSEHFTimeline: async (eaId, seType) => {
```

Comment:
The first parameter should be called `stateId`

Reply:
Done